index.html

```diff
@@ -566,8 +587,134 @@ <h1>Trust Model</h1>
 
     <section>
       <h1>Basic Concepts</h1>
+
+      <section>
+        <h2>Types</h2>
+
+        <p>
+Type information may be expressed via the <a>type</a> property:
+        </p>
+        <dl>
+          <dt><dfn>type</dfn></dt>
+          <dd>
+The value of this property MUST be one or more URIs.
+If more than one URI is provided, the array MUST be interpreted as a set of
+values where order is not significant. Note that syntax such as JSON-LD terms
+MAY be used to ease developer usage. It is RECOMMENDED that dereferencing the
+URI results in a document containing machine-readable information about the
+type.
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of the type property">
+{
+  "id": "http://dmv.example.gov/credentials/3732",
+  <span class="highlight">"type": ["VerifiableCredential", "ProofOfAgeCredential"]</span>,
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "ageOver": 21
+  },
+  "proof": { ... }
+}
+        </pre>
+
+        <p>
+With respect to this specification, the following <a>type</a>s MUST be
+specified on the following objects:
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Concept</th>
+              <th>Type</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            <tr>
+              <td>
+A <a href="#credentials">credential</a> object containing a
+<a href="#proofs-aka-signatures">proof</a>
+              </td>
+              <td>
+<tt>"type": ["<code>VerifiableCredential</code>"]</tt>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+A <a href="#credentials">credential</a> object NOT containing a
+<a href="#proofs-aka-signatures">proof</a>
+              </td>
+              <td>
+<tt>"type": ["<code>UnverifiableCredential</code>"]</tt>
```

Comment:
I'm not a fan of this name but I don't have an alternative. This name really only exists because we had to adopt "VerifiableCredential" rather than "Credential". Otherwise a "Verifiable Credential" would just be a "Credential" with a "proof" on it -- with no need to distinguish its type in this way. You can make a "Credential" verifiable by simply attaching a proof to it. As it stands now, to make an "UnverifiableCredential" verifiable, you'd have to *remove* its type, add a type of "VerifiableCredential" and add a proof.

This seems odd to me but I have nothing actionable to say that I think would be accepted. Likely unacceptable options are:

1. Rename "VerifiableCredential" to "Credential" and just use that type as I mentioned above.
2. Rename "UnverifiableCredential" to "Credential".
3. Remove type of "UnverifiableCredential" and leave it untyped.
4. Change "UnverifiableCredential" to "Assertion" and introduce further vernacular/confusion.

Option 4 would be to say that a simple "Assertion" stops being a mere assertion and becomes a "VerifiableCredential" once you attach proof to it. However, that, I believe this may further confuse the meaning of "proof". "Proof" does not prove that some statements are true, but rather the authenticity of the issuer.

That means that what we're really talking about here is more like "Gossip" or "Hearsay" or "Allegation". I *heard* that so and so said X about Y. I don't have any proof that they did.

Reply:
I do not support introducing 'unverifiable credential'. As pointed out in #178, credentials MUST always be verifiable since users are not trusted (regardless of whether they have been authenticated or not.)

Reply:
Fixed in PR #177.